src/application/main.cpp

```diff
@@ -273,6 +232,8 @@ int main(int argc, char **argv)
 
     std::cout << "PopSift version: " << POPSIFT_VERSION_STRING << std::endl;
 
+    config.setDescMode( popsift::Config::VLFeat_Desc );
+
```

Comment:
not a problem with your changes but it would be better to add an `else` branch in the check for the file to exist at line 246 below:
```c++
    if( boost::filesystem::exists( inputFile ) ) {
        if( boost::filesystem::is_directory( inputFile ) ) {
            cout << "BOOST " << inputFile << " is directory" << endl;
            collectFilenames( inputFiles, inputFile );
            if( inputFiles.empty() ) {
                cerr << "No files in directory, nothing to do" << endl;
                return EXIT_SUCCESS;
            }
        } else if( boost::filesystem::is_regular_file( inputFile ) ) {
            inputFiles.push_back( inputFile );
        } else {
            cout << "Input file is neither regular file nor directory, nothing to do" << endl;
            return EXIT_FAILURE;
        }
    }
+    else 
+    {
+        cout << "Input file/directory does not exist" << endl;
+        return EXIT_FAILURE;
+    }
```
so that the program exits if the input does not exist